apache/superset
# chore: Upgrades Storybook from 6.4.19 to 6.4.22

### SUMMARY
Upgrades Storybook from `6.4.19` to `6.4.22` to resolve a problem reported in https://github.com/storybookjs/storybook/issues/10887 and fixed by https://github.com/storybookjs/storybook/pull/17875

### TESTING INSTRUCTIONS
- Open the Storybook
- Make sure all the stories are working

### ADDITIONAL INFORMATION
